### DISCUSSION TITLE
can not grant access to the microphone

### DISCUSSION THREAD

**Discussion by egeruzz2424** [stoatchat/discussions#824] [Q&A]
https://github.com/user-attachments/assets/dc2062f7-0870-4b01-8f6d-0f9ee15c231a

**Comment by 52rpm** [comment]
same problem

**Comment by delidolu1adam** [comment]
Yukarıdaki bildirim mesajını okumadınız mı? :)

Şu anda istemciyi ve ses sunucusunu sıfırdan/yeniden oluşturuyoruz.
Eski ses çoğu durumda işe yarayabilir. Ancak bazı senaryolarda bağlantı kurulamayabilir ve/veya açıklanamayan garip davranışlar sergileyebilir.
